sdk/core/src/main/java/com/klaviyo/core/networking/KlaviyoNetworkMonitor.kt

```diff
@@ -39,11 +39,6 @@ internal object KlaviyoNetworkMonitor : NetworkMonitor {
 
         override fun onUnavailable() = broadcastNetworkChange()
 
-        override fun onCapabilitiesChanged(
-            network: Network,
-            networkCapabilities: NetworkCapabilities
-        ) = broadcastNetworkChange()
-
         override fun onLinkPropertiesChanged(
```

Comment:
is `onLinkPropertiesChanged` needed either?

Reply:
https://developer.android.com/reference/android/net/LinkProperties
Maybe more granular than we need, though maybe not as flappy as capabilities at least

Reply:
Yeah, actually something like a proxy change would be a good event to trigger the network queue to flush.